geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SimpleFeatureTypes.scala

```diff
@@ -102,6 +102,10 @@ object SimpleFeatureTypes {
 
     case t if specParser.parse(specParser.listType, t).successful => ListAttributeSpec(conf)
     case t if specParser.parse(specParser.mapType, t).successful  => MapAttributeSpec(conf)
+
+    case default => throw new IllegalArgumentException(s"Type $default is not a supported Simple Feature attribute type, "
+      + s"valid types are: ${simpleTypeMap.keys.toList.sorted}, ${geometryTypeMap.keys.toList.sorted}, "
+      + "or specs of the type \"id:Integer:opt1=v1:opt2=v2,*geom:Geometry:srid=4326,ct:List[String]:index=true,mt:Map[String,Double]:index=false\"")
```

Comment:
could we change this last line to just be `List[?], Map[?,?]` ? that seems more readable to me.